src/billing-reports/empty-state/empty-state-comingsoon.component.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { EmptyDataIllustration } from './empty-data-illustration.component';
+import { Trans, useTranslation } from 'react-i18next';
+import { Button, Tile, Layer } from '@carbon/react';
+import styles from './empty-state.scss';
+
+interface EmptyStateProps {
+  headerTitle: string;
+  displayText?: string;
+  launchForm?: () => void;
+  launchFormComponent?: any;
+  showLaunchLink?: boolean;
+}
+
+export const EmptyStateComingSoon: React.FC<EmptyStateProps> = ({ headerTitle, displayText }) => {
```

Comment:
@lucyjemutai We don't need this, there already exists an empty state from the openmrs framework, we should use that rather than duplicate